apps/condo/domains/common/utils/helpers.ts

```diff
@@ -12,6 +14,23 @@ export const formatPhone = (phone?: string): string => {
     */
 }
 
+/**
+ * Formats raw timestamp string into human readable form, depending of what year it represents:
```

Comment:
redurant comment

Reply:
It's a not mine comment to `formatPhone`, moved it to top of the function in `ccf2117b`.